ecl/regress/regress.sh

```diff
@@ -39,7 +39,7 @@ diff=
 query=
 valgrind=
 np=`getconf _NPROCESSORS_ONLN`
-export ECLCC_ECLINCLUDE_PATH=
+#export ECLCC_ECLINCLUDE_PATH=
```

Comment:
accidentally included.

Reply:
Is there a good way to fix this permanently? At present I need to comment this line out to run regress.sh successfully against runtime suite, and add it back in to run against public compiler suite...

Reply:
My script that runs this script for the runtime regression suite includes the necessary -I commands.  I suspect another fix would be to add --nostdinc when running the public compiler suite.